Rename tests end with BoltKitTest to BoltKitIT

bigmontz opened the pull request:
The BoltKitTests depends on the TeamCity environment with the new test environment be running to not be skipped,
so it should be moved to the integration tests phase avoiding being always skipped during the unit tests phase.

Rename it with the suffix `BoltKitIT` does this job.

michael-simons reviewed (approved):
LGTM. I guess you needed to explicitly include the `*IT.java` and `*ITCase.java` because one entry would overwrite all the others, right?

bigmontz commented:
> LGTM. I guess you needed to explicitly include the `*IT.java` and `*ITCase.java` because one entry would overwrite all the others, right?

Exactly

2hdddg commented:
Would it be a bigger but more safe change to append IT to the boltkit tests?
We need to be sure that we dont lose any tests in the process (the test count in Teamcity can be used to verifiy)

bigmontz commented:
> Would it be a bigger but more safe change to append IT to the boltkit tests?
> We need to be sure that we dont lose any tests in the process (the test count in Teamcity can be used to verifiy)

I could named `BoltKitIT` instead of `BoltKitTest`. I also get a bit more explicit the intention in the code.

bigmontz committed:
Rename tests end with BoltKitTest to the BoltKitIT

The BoltKitTests depends on the TeamCity environment with the new test environment be running to not be skipped,
so it should be moved to the integration tests phase avoiding being always skipped during the unit tests phase.

Rename it to `BoltKitIT` does this job.
